src/modules/Gapi.ts

```diff
@@ -0,0 +1,75 @@
+declare global {
+  interface Window {
+    gapi: any; 
+  }
+}
+
+const {
+    REACT_APP_GOOGLE_OAUTH_CLIENT_ID,
+    REACT_APP_GOOGLE_YOUTUBE_API_KEY,
+    REACT_APP_GOOGLE_AUTH_SCOPE_URL,
+    REACT_APP_GOOGLE_YOUTUBE_API_URL
+} = process.env;
+
+class GoogleApi {
+    
+    gapi: any;
+
+    constructor() {
+        const { gapi } = window;
+        this.gapi = gapi;
+    }
+
+    loadGapiClientAuth2 = async () => {
```

Comment:
A lot of these methods can be `static`

Reply:
Sure. Let me see what I can do after implementing the new library.